### PR TITLE
PR：部分优化

### PR DESCRIPTION
**Pull request by PythonSmall-Q**
对配置页面增加解释，提交界面优化提示等

### CODE REVIEW

File: XMOJ.user.js [hunk 1]
```diff
@@ -1009,7 +1008,7 @@ if (document.querySelector("#navbar") != null) {
                     }
                     let OpenUnsolvedButton = document.createElement("button");
                     OpenUnsolvedButton.className = "btn btn-outline-secondary";
-                    OpenUnsolvedButton.innerText = "打开未解决题目";
+                    OpenUnsolvedButton.innerText = "打开未解决题目（OI模式下请手动打开题目）";
```
Comment (langningchen) [inline XMOJ.user.js:1011]: I think you misunderstood the meaning of "Unsolved problem". Unsolved problem including problems that not judged or ac and problems that not submitted.

**Review by langningchen — APPROVED** [review]
I think this is ok

**Comment by langningchen** [comment]
If you think there is no problem, I will merge the pull request.

**Review by PythonSmall-Q — COMMENTED** [review]
think there is no prob try deploying/

**Pull request merged**